inference-engine/src/mkldnn_plugin/nodes/mkldnn_multiclass_nms.cpp

```diff
@@ -0,0 +1,566 @@
+// Copyright (C) 2018-2021 Intel Corporation
```

Comment:
apply ```MKLDNNMatrixNmsNode``` comments to ```MKLDNNMultiClassNmsNode``` as well please

Reply:
For Multiclass NMS  Node: Copyright will change to "Copyright (C) 2021", some unused including will be deleted, "checkOutput()" function will be changed to "checkPrecision()" in next commit.